packages/demos/src/feature-app-in-feature-app/integrator.node.tsx

```diff
@@ -0,0 +1,30 @@
+import {FeatureAppManager, FeatureServiceRegistry} from '@feature-hub/core';
+import {loadCommonJsModule} from '@feature-hub/module-loader-commonjs';
+import {FeatureAppLoader, FeatureHubContextProvider} from '@feature-hub/react';
+import * as React from 'react';
+import * as ReactDOM from 'react-dom/server';
+import {AppRendererOptions, AppRendererResult} from '../start-server';
+
+export default async function renderApp({
+  port
+}: AppRendererOptions): Promise<AppRendererResult> {
+  const featureAppNodeUrl = `http://localhost:${port}/feature-app.commonjs.js`;
+  const featureServiceRegistry = new FeatureServiceRegistry();
+
+  const featureAppManager = new FeatureAppManager(featureServiceRegistry, {
+    moduleLoader: loadCommonJsModule
+  });
+
+  await featureAppManager.preloadFeatureApp(featureAppNodeUrl);
```

Comment:
What am I not getting here - why do we preload on the server?

Reply:
Because we don’t use the Async SSR Manager here (for simplicity).

Reply:
Understood. I'm just afraid that this is confusing for someone reading this demo...

Reply:
will add a comment...